openslides_backend/action/motion_category/create_update_delete.py

```diff
@@ -0,0 +1,21 @@
+from ...models.motion_category import MotionCategory
+from ..action import register_action_set
+from ..action_set import ActionSet
+from ..default_schema import DefaultSchema
+
+
+@register_action_set("motion_category")
+class MotionCategoryActionSet(ActionSet):
+    """
+    Actions to create, update and delete motion categories.
+    """
+
+    model = MotionCategory()
+    create_schema = DefaultSchema(MotionCategory()).get_create_schema(
+        properties=["name", "prefix", "weight", "level", "meeting_id", "parent_id"],
```

Comment:
`level` is a calculated field: This should never be written by the user. `weight` is not provided by the user: it is just set to a large value (like 1000), so the category will be at the bottom (presumably, the nicest way would be to get the max weight and use `max_weight+1` , but this is too much effort)

Reply:
parent_id, weight and level removed from the properties.